ProjectTemplates/templates/StarterWeb-CSharp/Program.cs

```diff
@@ -0,0 +1,26 @@
+using System;
```

Comment:
do we need license info on these files?

Reply:
That's a good question
however these are the projects that customers will create, and they won't necessarilly want to have open source code with the MIT license :)
so, no!